cli/cluster/instances.go

```diff
@@ -141,14 +150,15 @@ func ConnectToInstance(instanceName string, ctx *context.Ctx) (*connector.Conn,
 	return conn, nil
 }
 
-func getRunningInstances(instancesConf *InstancesConf, ctx *context.Ctx) []string {
-	var runningInstancesNames []string
-	for instanceName := range *instancesConf {
+// Returns URI -> instance name map of running instances
```

Comment:
https://go.dev/blog/godoc 
`Notice this comment is a complete sentence that begins with the name of the element it describes.`